imap/http_caldav.c

```diff
@@ -610,7 +610,12 @@ static struct meth_params caldav_params = {
       { NS_CALDAV, "calendar-data", &caldav_import } },
     { CALDAV_SUPP_DATA, &caldav_put },
     { 0, caldav_props },                        /* Allow infinite depth */
-    caldav_reports
+    caldav_reports,
+    { offsetof(struct mboxname_counters, caldavmodseq),
```

Comment:
This is really clever and I hope portable!  It probably needs a comment somewhere explaining how and why it's used,.

Reply:
libical uses stuff like this, so I assume its portable